purchase_invoice_plan/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+- Create new purchase quotation as per normal process
+- Select option "Use Invoice Plan", a new Invoice Plan tab will appear
+- Click on "=> Create Invoice Plan" link to open invoice planning wizard
+- Do plan for number of installment, start date and interval
+- Double check that each installment has correct plan percentage
+- After confirm purchases order, we have new option to "Create Bill by Plan"
```

Comment:
```suggestion
- Confirm Order > Receive Products, see new option to "Create Bill by Plan"
```

Reply:
this is my mistake, I config product purchase policy is `On received quantities`
it should be `On ordered quantities`